extension/src/popup/views/AccountCreator/styles.scss

```diff
@@ -0,0 +1,40 @@
+.AccountCreator {
+  margin-bottom: 1rem;
+
+  &--header {
+    font-size: 2.125rem;
+    line-height: 1.2rem;
+    font-weight: normal;
```

Comment:
there's actually a var for this `--font-weight-normal`

Reply:
da2065e